oca_dependencies.txt

```diff
@@ -1 +1 @@
-partner-contact https://github.com/OCA/partner-contact 8.0
+partner-contact https://github.com/OCA/partner-contact 10.0
```

Comment:
better leave out the url and the branch name altogether, then the maintainer tools will pick the current version's branch